src/locale/languages/es.js

```diff
@@ -90,6 +90,7 @@ module.exports = {
   'No contributors found': 'No se encontraron contribuidores',
   'No description provided.': 'No se ha proporcionado ninguna descripción.',
   'No issues': 'No hay issues',
+  'No members found': '',
```

Comment:
friendly ping @alejandronanez 
Can you translate a new phrase?

Reply:
:wave: @lex111,

I think this could work.

```suggestion
'No members found': 'No hay miembros'
```

Reply:
That's right, this is a new feature of GitHub, thanks!